scaladoc/src/dotty/tools/scaladoc/site/FrontMatterRenderer.scala

```diff
@@ -0,0 +1,22 @@
+package dotty.tools.scaladoc
+package site
+
+
+/**
+ * Object for rendering yaml front-matter for preprocessed markdowns.
+ *
+ */
+object FrontMatterRenderer:
+  def render(properties: Map[String, Object]): String =
+    val propsToInsert = Seq("title", "type", "num", "previous-page", "next-page")
```

Comment:
This seems to be very specific to docs.scala-lang.org.

Reply:
I think we won't need this transformer at all if we drop completely generating static site by scaladoc